binder/views.py

```diff
@@ -1396,9 +1396,19 @@ def store_m2m_field(obj, field, value, request):
 
 		try:
 			obj.save()
+			if obj.pk is None:
+				# Something went wrong while creating/updating the object, and no validation
+				# errors were raised, something must be wrong...
+				raise RuntimeError('Error while saving object')
```

Comment:
Since you throw a `RuntimeError` I expect this is just to make sure that this never happens?
In that case an assert statement might be clearer here.

Reply:
Changed.